conda/recipes/cudf/meta.yaml

```diff
@@ -20,16 +20,18 @@ build:
 
 requirements:
   build:
+    - protobuf
     - python
     - cython >=0.29,<0.30
     - setuptools
     - numba >=0.49.0
-    - dlpack
+    - dlpackx
```

Comment:
Is this intentional? I can't find any reference to dlpackx online.

Reply:
Sorry, I forgot to merge upstream changes that fixed this.